src/app/components/views/weather-view/weather-view.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, OnInit } from '@angular/core';
+import { WeatherAlertResponse } from 'src/app/models/weather-alert.model';
```

Comment:
local and package imports need separated

Reply:
done